servers/auth/conf

```diff
@@ -0,0 +1,13 @@
+rule.all.action=auth
+rule.all.rule=Path(`/os`)
+
+rule.noauth.action=allow
+rule.noauth.rule=Path(`/public`)
+
+rule.onlyu1.action=auth
+rule.onlyu1.rule=Path(`/user1`)
```

Comment:
Even after logging in as user2, the `localhost/user1` is accessible without logging out of user2. The same is true for user1.